resnet/README.md

```diff
@@ -22,107 +22,181 @@
+```bash
+composer main.py yamls/resnet50.yaml recipe_name=mild
+```
 
-      ```
-      python /tmp/create_ffcv_datasets.py --dataset imagenet --split train --datadir /tmp/ImageNet/
-      python /tmp/create_ffcv_datasets.py --dataset imagenet --split val --datadir /tmp/ImageNet/
-      ```
+---
+**NOTE**
 
-      **Note:** The helper script output the FFCV formatted dataset files to `/tmp/imagenet_train.ffcv` and `/tmp/imagenet_val.ffcv` 
-      for the training and validation data, respectively.
+The throughput of ResNet50 and smaller models are dataloader bottlenecked when training with our recipes on 8x NVIDIA A100s. This means the model's throughput is limited to the speed the data can be loaded. One way to alleviate this bottleneck is to use the [FFCV dataloader format](https://github.com/libffcv/ffcv). This [tutorial](https://docs.mosaicml.com/en/v0.11.0/examples/ffcv_dataloaders.html) walks you through creating your FFCV dataloader. We also have example code for an ImageNet FFCV dataloader [here](https://github.com/mosaicml/composer/blob/a0f441537008a1ef2678f1474f3cd5519deb80fa/composer/datasets/imagenet.py#L179).
 
-1. Launch the training run.
+Our best results use FFCV, so an FFCV version of ImageNet is required to duplicate our results.
 
-   ```
-   composer -n {num_gpus} train.py -f {recipe_yaml_path} --scale_schedule_ratio {scale_schedule_ratio}
-   ```
+---
+# Saving and Loading checkpoints
 
-   Replace `num_gpus`, `recipe_yaml_path` and `scale_schedule_ratio` with the total number of GPU's, the recipe configuration, and the scale schedule ratio we determined in the previous section for the desired run, respectively.
+At the bottom of `yamls/resnet50.yaml`, we provide arguments for saving and loading model weights. Please specify the `save_folder` or `load_path`arugments if you need to save or load checkpoints!
 
-   **Note:** The `Mild` and `Medium` recipes assume the training and validation data is stored at the `/tmp/imagenet_train.ffcv` and `/tmp/imagenet_val.ffcv` paths while the `Hot` recipe assumes the original ImageNet dataset is stored at the `/tmp/ImageNet` path.  The default dataset paths can be overridden, please run `composer -n {num_gpus} train.py -f {recipe_yaml_path} --help` for more detailed recipe specific configuration information.
-   
-   Example:
-   
-   ```
-   composer -n 8 train.py -f recipes/resnet50_mild.yaml --scale_schedule_ratio 0.32
-   ```
+# On memory constraints
+In previous blogs ([1](https://www.mosaicml.com/blog/farewell-oom), [2](https://www.mosaicml.com/blog/billion-parameter-gpt-training-made-easy))
+we demonstrated Auto Grad Accum, which allows Composer to determine `grad_accum` on its own. This means the same configuration can be run on different hardware or on a fewer number of devices without having to manually adjust the gradient accumulation! We have done extensive testing on this feature, but if there are any issues you can manually set `grad_accum` to your desired value.
```

Comment:
I doubt most users know what `grad_accum` is. We need a sentence explaining it if we're going to talk about (which maybe we don't need to?)

Reply:
I just took this from `llm`, I'm fine with deleting

Reply:
I deleted it